src/translations/en.json

```diff
@@ -143,6 +143,8 @@
   "CustomExtendedDataField.required": "Required",
   "CustomExtendedDataField.numberTooSmall": "The number must be larger than or equal to {min}",
   "CustomExtendedDataField.numberTooBig": "The number must be less than or equal to {max}",
+  "YoutubeVideoUrlDataField.placeholder": "Add Youtube URL",
+  "YoutubeVideoUrlDataField.notValidURL": "The URL you have entered is not a valid Youtube URL",
```

Comment:
Rename the keys: 
- "CustomExtendedDataField.placeholderYoutubeVideoURL"
- "CustomExtendedDataField.notValidYoutubeVideoURL"

And then sort these to alphabetical order.